tests/protocol/test_ipc.py

```diff
@@ -102,3 +107,38 @@ def get_context(self) -> None:
     args = [1, 2, 3, 4]
     result = await client_protocol.call(Echo(args))
     assert args == result
+
+
+async def test_log_transport(caplog):
+
```

Comment:
I think that every test case needs a description on what it does. The naming of the tests + variables is not enough imho.

Inconsistencies in terminology do not help either: shipper vs drainer